### PR TITLE
[bitnami/postgresql-ha] Add relabelings / metricRelabelings to ServiceMonitor

### PR DESCRIPTION
**Pull request by RichieSams**
Straight pass-through with template support

- [ x ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ x ] Variables are documented in the README.md
- [ x ] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

### CODE REVIEW

File: bitnami/postgresql-ha/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql-ha
-version: 2.3.1
+version: 2.3.2
```
Comment (juan131) [inline bitnami/postgresql-ha/Chart.yaml:3]: We should bump the minor version when adding a new feature/parameter to the chart

File: bitnami/postgresql-ha/README.md [hunk 2]
```diff
@@ -173,6 +173,8 @@ The following table lists the configurable parameters of the PostgreSQL HA chart
...
 | `metrics.serviceMonitor.interval`              | How frequently to scrape metrics (use by default, falling back to Prometheus' default)                                                                               | `nil`                                                        |
 | `metrics.serviceMonitor.selector`              | Default to kube-prometheus install (CoreOS recommended), but should be set according to Prometheus install                                                           | `{prometheus: "kube-prometheus"}`                            |
+| `metrics.serviceMonitor.relabelings`           | ServiceMonitor relabelings                                                                                                                                           | `[]`                                                         |
+| `metrics.serviceMonitor.metricRelabelings`     | ServiceMonitor metricRelabelings                                                                                                                                     | `[]`                                                         |
```
Comment (juan131) [inline bitnami/postgresql-ha/README.md:177]: You're evaluating both parameters a template. Therefore, please reflect that:
```diff
- | `metrics.serviceMonitor.relabelings` | ServiceMonitor relabelings   | `[]`                                         |
+ | `metrics.serviceMonitor.relabelings` | ServiceMonitor relabelings   | `[]` (The value is evaluated as a template). |
```
Apply the same suggestion to both parameters

File: bitnami/postgresql-ha/values.yaml [hunk 3]
```diff
@@ -543,6 +543,16 @@ metrics:
...
+    
+    ## RelabelConfigs to apply to samples before scraping
+    ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#relabelconfig
+    ##
+    # relabelings: []
+
+    ## MetricRelabelConfigs to apply to samples before ingestion
+    ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#relabelconfig
+    ##
+    # metricRelabelings: []
```
Comment (juan131) [inline bitnami/postgresql-ha/values.yaml:555]: Could you also update the **values-production.yaml** with the same changes?

**Comment by RichieSams** [comment]
Fixed. Thanks for the feedback!

**Comment by juan131** [comment]
Waiting for CI/CD to test your changes and update image tags

**Pull request merged**